tasks/framework-tools/lib/framework.mjs

```diff
@@ -170,15 +169,15 @@ export function buildPackages(packages = frameworkPkgJsonFiles()) {
     {
       shell: true,
       stdio: 'inherit',
-      cwd: path.resolve(__dirname, '../../../../'),
+      cwd: path.resolve(__dirname, '../../../'),
```

Comment:
Same here, not sure why this path has changed

Reply:
I did move it actually; the filepath used to be `tasks/framework-tools/src/lib/framework.mjs`.